main/lsp/LSPIndexer.cc

```diff
@@ -214,6 +214,15 @@ LSPIndexer::getTypecheckingPathInternal(const vector<shared_ptr<core::File>> &ch
     result.path = TypecheckingPath::Fast;
     logger.debug("Taking fast path");
     timeit.setTag("path_chosen", "fast");
+
+    // Using the incremental name indicates that we changed a symbol that affects an unrelated file, and thus traversed
+    // the file table to determine if we can still take the fast path.
+    if (result.files.useIncrementalNamer) {
+        timeit.setTag("name_hashes_changed", "true");
+    } else {
+        timeit.setTag("name_hashes_changed", "false");
+    }
```

Comment:
I think if you use this opportunity to set `"path_chosen", "fast+incremental_namer"`, then our dashboard charts, including the hubble charts, will magically start computing what we want.

Reply:
(I'm fine either way)

Reply:
I like that!